package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tocca",
-  "version": "2.0.9",
+  "version": "2.1.0",
```

Comment:
please avoid changes in the `package.json` file

Reply:
I'll revert package number.